apps/test/unit/code-studio/pd/professional_learning_landing/csPrinciplesAndDiscoveriesSectionTest.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import CsPrinciplesAndDiscoveriesSection from '@cdo/apps/code-studio/pd/professional_learning_landing/csPrinciplesAndDiscoveriesSection';
+import {expect} from 'chai';
+
+describe("CS Principles and Discoveries Section", () => {
+  const generateSection = (sectionProps = []) => {
+    return mount(
+      <CsPrinciplesAndDiscoveriesSection
+        {...sectionProps}
+      />
+    );
+  };
+
+  it("should show a survey for a user that needs to take a survey", () => {
+    const section = generateSection({coursesTaught: ['CS Principles'], lastWorkshopSurveyUrl: 'url'});
+
+    expect(section.find('#cspStartSurvey')).to.have.length(1);
+    expect(section.find('#cspThanks')).to.have.length(0);
```

Comment:
No - this is valid. The workshop has ended so there's a survey that needs to be taken, hence we'd expect #cspStartSurvey. If the workshop was ended but the survey was already completed, we'd expect #cspThanks